views/footer.ejs

```diff
@@ -1 +1 @@
-<p class="text-center" style="font-size:smaller;">© Your IT, <a href="#" target="_blank">Your IT</a></p> 
\ No newline at end of file
+<p class="text-center" style="font-size:smaller;">Last Updated: <span class="fileUpdate"><em>Unknown</em></span><br /><br />&copy;2016 Your IT</p> 
```

Comment:
`fileUpdate` should be an id stead of a class since there is only one such element